apps/openmw/mwmechanics/character.cpp

```diff
@@ -1874,7 +1874,7 @@ void CharacterController::update(float duration)
                                          : (sneak ? CharState_SneakBack
                                                   : (isrunning ? CharState_RunBack : CharState_WalkBack)));
             }
-            else if(rot.z() != 0.0f && !inwater && !sneak && !(mPtr == getPlayer() && MWBase::Environment::get().getWorld()->isFirstPerson()))
+            else if(rot.z() != 0.0f && !sneak && !(mPtr == getPlayer() && MWBase::Environment::get().getWorld()->isFirstPerson()))
```

Comment:
I am not sure about isFirstPerson() check. Is it really needed? There is an only usage of this function in the character manager.

Reply:
Don't recall exactly but the isFirstPerson() was probably added for a reason. Try 'git blame'. Also check for marksman aiming and weapon idle animations - these should not interrupt by turning.